workflow_tests/deployBlocker.test.ts

```diff
@@ -1,12 +1,14 @@
-const path = require('path');
-const kieMockGithub = require('@kie/mock-github');
-const utils = require('./utils/utils');
-const assertions = require('./assertions/deployBlockerAssertions');
-const mocks = require('./mocks/deployBlockerMocks');
-const ExtendedAct = require('./utils/ExtendedAct').default;
+/* eslint-disable @typescript-eslint/naming-convention */
+import {MockGithub} from '@kie/mock-github';
+import path from 'path';
+import assertions from './assertions/deployBlockerAssertions';
+import * as mocks from './mocks/deployBlockerMocks';
```

Comment:
```suggestion
import mocks from './mocks/deployBlockerMocks';
```

Merge with `main` first